app_dart/test/request_handlers/file_flaky_issue_and_pr_test.dart

```diff
@@ -332,31 +332,6 @@ void main() {
       expect(result['Status'], 'success');
     });
 
-    test('Can file issue but not pr for test not found in ci.yaml', () async {
```

Comment:
This test removal LGTM now that flutter/infra is generated from ci.yaml, but Keyong should approve as well

Reply:
Somehow I remembered leaving a comment to ack the test removal... anyway, this LGTM as now the .ci.yaml is the source of truth.